Uplift/Views/DropdownView.swift

```diff
@@ -64,10 +67,19 @@ class DropdownView: UIView {
         self.headerView.isUserInteractionEnabled = true
         addSubview(self.headerView)
 
+        self.arrowImageView = arrowImageView
+        if let arrowView = self.arrowImageView {
+            if self.headerView.subviews.contains(arrowView) {
+                arrowImageViewNeedsConstraints = false
+            } else {
+                self.headerView.addSubview(arrowView)
+            }
+        }
```

Comment:
why do we need to check this? shouldn't the headerView always have an arrowImageView which we just hardcode to be the image for the arrow?

Reply:
In case the `headerView` for whatever reason does not already have an `arrowImageView` as a subview. It makes setting up a dropdown view less complicated.

Reply:
we shouldn't need this if we just let the user handle including//not including the `arrowImageView` in the `headerView` that is passed in